<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CTKJV5C" failures="1" tests="4" name="Pet Test" time="24.512" errors="0" timestamp="2025-06-30T12:15:57 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="9.176"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="5.429"/>
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="7.584"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserByName" time="1.25">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testDeleteUserByName(UserTests.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- Pet Test -->
